src/main/webapp/app/exercises/programming/manage/update/programming-exercise-plans-and-repositories-preview.component.html

```diff
@@ -1,67 +0,0 @@
-@if (programmingExercise && programmingExercise.shortName) {
```

Comment:
This component is now under `shared/build-details` as we want to display it on the edit and details page.

It is also split into two components now:
`programming-exercise-build-plan-checkout-directories.component` and `programming-exercise-repository-and-build-plan-details.component`